lib/themes/dosomething/paraneue_dosomething/js/components/CampaignSlide.js

```diff
@@ -35,20 +41,22 @@ class CampaignSlide extends React.Component {
           <div className="wrapper">
             {this.renderHeader()}
 
-            {this.campaigns.map(campaign => {
-              // Fix for local files / data missing by preventing null error
-              const image_uri = campaign.cover_image.default != null ? campaign.cover_image.default.sizes.landscape.uri : '';
+            <div className="gallery -triad">
+              {this.state.campaigns.map(campaign => {
+                // Fix for local files / data missing by preventing null error
+                const image_uri = campaign.cover_image.default != null ? campaign.cover_image.default.sizes.landscape.uri : '';
```

Comment:
Can we use [Lodash's `get`](https://lodash.com/docs/4.16.0#get) for this? The ternary is kinda ugly to read and potentially brittle if other keys in the chain are missing.